python_modules/libraries/dagster-twilio/dagster_twilio/resources.py

```diff
@@ -1,13 +1,23 @@
-from dagster import Field, StringSource, resource
+from dagster import ConfigurableResource, resource
+from dagster._config.structured_config import infer_schema_from_config_class
+from dagster._core.execution.context.init import InitResourceContext
+from pydantic import Field as PyField
 from twilio.rest import Client
 
 
+class TwilioResource(ConfigurableResource[Client]):
```

Comment:
Hmm in terms of norms here is it not more clear to inherit from `ConfigurableResourceFactory` directly?

Reply:
Yes, this was an early port that predated `ConfigurableResourceFactory`. Updated!